addon/mixins/route.js

```diff
@@ -217,6 +217,10 @@ export default Ember.Mixin.create({
     return false;
   },
 
+  updateInfinityModel(model, infinityModel) {
+    model.pushObjects(infinityModel.get('content'));
+  },
+
```

Comment:
:+1:, I really like that. Will make that change and potentially update the README too.